collin/index.html

```diff
@@ -0,0 +1,31 @@
+<!-- <!DOCTYPE html> -->
+<html lang="en-US">
+
+<head>
+	<meta charset="utf-8">
+	<title>Got Jokes</title>
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+	<h1 id="title">Riddle me this</h1>
+	<main class="joke-body">
+		<div class="joke-setup">
+			<p>Why did Nathan cross the road?</p>
+		</div>
+		<div class="punchline-box">
+			<img src="https://tinyurl.com/y4lzbnfs" alt="hover over me">
```

Comment:
![image](https://user-images.githubusercontent.com/13723156/102022674-f3621680-3d4d-11eb-8107-7e25f87e5cab.png)

It looks like the `src` attribute is missing a filetype extension. And make sure that your `alt` attribute is descriptive of what the image _looks_ like, not what it _does_. "hover over me" is fairly meaningless for a keyboard/screen-reader user.